codersdk/users.go

```diff
@@ -189,12 +189,23 @@ type ValidateUserPasswordResponse struct {
 	Details string `json:"details"`
 }
 
+// TerminalFontName is the name of supported terminal font
+type TerminalFontName string
+
+const (
+	TerminalFontUnknown     TerminalFontName = ""
+	TerminalFontIbmPlexMono TerminalFontName = "ibm-plex-mono"
```

Comment:
```suggestion
	TerminalFontIBMPlexMono TerminalFontName = "ibm-plex-mono"
```
Suggestion: Initialism.

Reply:
Fixed